transport/transport-runtime/src/androidTest/java/com/google/android/datatransport/runtime/UploaderIntegrationTest.java

```diff
@@ -108,14 +109,12 @@ public void uploader_transientError_shouldReschedule() {
     transport.send(stringEvent);
     verify(mockBackend, times(2))
         .send(eq(BackendRequest.create(Collections.singletonList(expectedEvent))));
-    verify(spyScheduler, times(1)).schedule(any(), eq(2));
+    verify(spyScheduler, times(1)).schedule(any(), eq(2), anyBoolean());
     Iterable<PersistedEvent> eventList = store.loadBatch(transportContext);
     assertThat(eventList).isNotEmpty();
     for (PersistedEvent persistedEvent : eventList) {
       assertThat(persistedEvent.getEvent()).isEqualTo(expectedEvent);
     }
-
-    assertThat(store.getNextCallTime(transportContext)).isEqualTo(0);
```

Comment:
This was removed because we now will store the next call time even during errors (to the current time + the max nextRequestCallTime seen by any previous successful uploads.) In the case where there are no previous uploads, although the value is different, I think the functionality should be the same since the difference is just 0 vs the current time.